src/Avalonia.X11/X11Clipboard.cs

```diff
@@ -287,10 +292,13 @@ private Task StoreAtomsInClipboardManager(IDataObject data)
             return Task.CompletedTask;
         }
 
-        public Task SetTextAsync(string text)
+        public Task SetTextAsync(string? text)
         {
             var data = new DataObject();
-            data.Set(DataFormats.Text, text);
+
+            if (text is not null)
```

Comment:
Need to check how we've previously handled this scenario, I think we were responding with an empty string to selection query